src/Swarm/Language/Parse.hs

```diff
@@ -377,8 +378,8 @@ runParser p t = first (from . errorBundlePretty) (parse (runReaderT p DisallowAn
 runParserTH :: (Monad m, MonadFail m) => (String, Int, Int) -> Parser a -> String -> m a
 runParserTH (file, line, col) p s =
   case snd (runParser' (runReaderT (fully p) AllowAntiquoting) initState) of
-    Left err -> fail $ errorBundlePretty err
-    Right e -> return e
+    Prelude.Left err -> fail $ errorBundlePretty err
+    Prelude.Right e -> return e
```

Comment:
I assume this is leftover too, since you do not define new `Right`, right? :thinking:

Reply:
Another good catch!